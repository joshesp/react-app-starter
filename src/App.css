.title-app {
	font-size: 20px;
	color: dimgrey;
}
