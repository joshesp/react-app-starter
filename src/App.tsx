import React from "react"
import "./App.scss"
import Logo from "./assets/images/workflow.svg"
import { APP_TITLE } from "./constants/environment"

const App = () => {
	return (
		<div className="app">
			<img src={Logo} alt={APP_TITLE} width="60" />
			<h1 className="title-app">{APP_TITLE}</h1>
		</div>
	)
}

export default App
