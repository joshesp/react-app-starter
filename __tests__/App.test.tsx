import { render } from '@testing-library/react';
import React from 'react';
import App from "../src/App";

describe("App", () => {
	test("should equal to snapshot", () => {
		const {container, getByText } = render(<App />)
		expect(container).toMatchSnapshot()
	})
})
