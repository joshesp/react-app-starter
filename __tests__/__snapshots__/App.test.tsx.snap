// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should equal to snapshot 1`] = `
<div>
  <h1
    class="title-app"
  >
    My react app
  </h1>
</div>
`;
